lib/MetadataStore.js

```diff
@@ -231,39 +237,44 @@ MetadataStore.prototype = {
 
         /* 2. inserts the favicon into page_images, handles the special case, in which
          * the favicon already exsits */
-        let faviconBindings = this._getFaviconParameters(metaObject);
```

Comment:
This is just nesting the block inside an `if (metaObject.favicon_url) {`, sorry the diff is so bad